cephadm-preflight.yml

```diff
@@ -0,0 +1,89 @@
+---
+# This playbook configures the Ceph repository.
+# It also installs some prerequisites (podman, lvm2, chronyd, ...)
+#
+# Usage:
+#
+# ansible-playbook -i <inventory host file> cephadm-preflight.yml
+#
+# You can limit the execution to a set of hosts by using `--limit` option:
+#
+# ansible-playbook -i <inventory host file> cephadm-preflight.yml --limit <my_osd_group|my_node_name>
+#
+# You can override variables using `--extra-vars` parameter:
+#
+# ansible-playbook -i <inventory host file> cephadm-preflight.yml --extra-vars "ceph_origin=rhcs"
+#
+
+
+- hosts: all
+  become: true
```

Comment:
When `become` is explicitly defined this might work perfectly for users of CentOS setup via cloud-init who are implicitly given a non-root user, though when kickstarting bare-metal this likely won't be the case. Ansible already provides `--become*` flags for situations like this which could be documented above.

Reply:
Except if you can ssh nodes as `root` (which is not something you want to do IMO), is there a scenario where you could run this playbook without `become: true` or `--become`? Could you describe your use case?
Personally, I don't mind asking people to run with `--become` but in that case, why not explicitly defining it at the play level?

Reply:
> Except if you can ssh nodes as `root` (which is not something you want to do IMO)
I am (unfortunately) that type of person :smile: 

> Personally, I don't mind asking people to run with --become but in that case, why not explicitly defining it at the play level?
After thinking about this some more, I suppose that even if _someone_ (ie me) wanted to use `root`, the use of `become` has no tangible impact. I haven't personally read over how Ansible handles that internally, though leaving `become` present skips a parameter for non-`root` users... I think `become` is also making testing with Vagrant a bit easier.

Good points though, thanks for clarifying!